app/models/tag.rb

```diff
@@ -1194,4 +1194,15 @@ def update_tag_nominations(tag, deleted=false)
     TagNomination.where(tagname: tag.name).update_all(values)
   end
 
+  def only_case_changed?
+    new_normalized_name = normalized_without_unicode(self.name)
+    old_normalized_name = normalized_without_unicode(self.name_was)
+    (self.name.downcase == self.name_was.downcase) ||
+      (new_normalized_name == old_normalized_name)
+  end
+
+  # Produce an ASCII, downcased version of a string
+  def normalized_without_unicode(str)
+    str.mb_chars.unicode_normalize(:nfkd).gsub(/[^\x00-\x7F]/u,'').downcase.to_s
```

Comment:
Because the normalization form was changed, can you add tests for the new behavior, as well as notes for what QA should check?

Reply:
The behavior should be the same, and I believe the form is "normalize form kd" in both cases. It doesn't change any values, it just normalizes the strings for comparison, and we test the desired behavior in the tag model spec.

Reply:
Got it, I missed that `normalize` was deprecated in favor of `String#unicode_normalize`.